src/shadowbox/integration_test/test.sh

```diff
@@ -143,7 +147,7 @@ function cleanup() {
   # Verify we can't access the URL anymore after the key is deleted
   client_curl --insecure -X DELETE ${SB_API_URL}/access-keys/0 > /dev/null
   # Exit code 56 is "Connection reset by peer".
```

Comment:
Are we going to get connection reset? With the timeout change against probing, shouldn't an invalid key time out?
I think this also depend on how the go-shadowsocks2 client forwards the errors to the destination. Maybe something changed and a reset to the proxy is returning a clean close to curl.